daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-topics.md

```diff
@@ -0,0 +1,164 @@
+---
+type: docs
+title: "Azure Service Bus Topics"
+linkTitle: "Azure Service Bus Topics"
+description: "Detailed documentation on the Azure Service Bus Topics pubsub component"
+aliases:
+  - "/operations/components/setup-pubsub/supported-pubsub/setup-azure-servicebus-topics/"
+  - "/operations/components/setup-pubsub/supported-pubsub/setup-azure-servicebus/"
+---
+
+## Component format
+
+To setup Azure Service Bus Topics pubsub create a component of type `pubsub.azure.servicebus.topics`. See [this guide]({{< ref "howto-publish-subscribe.md#step-1-setup-the-pubsub-component" >}}) on how to create and apply a pubsub configuration.
+
+> This component uses [topics](https://learn.microsoft.com/azure/service-bus-messaging/service-bus-queues-topics-subscriptions) on Azure Service Bus. For queues, see the [Azure Service Bus Queues pubsub component]({{< ref "setup-azure-servicebus-queues" >}}).
+
+### Connection String Authentication
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: servicebus-pubsub
+spec:
+  type: pubsub.azure.servicebus.topics
+  version: v1
+  metadata:
+  - name: connectionString # Required when not using Azure Authentication.
+    value: "Endpoint=sb://{ServiceBusNamespace}.servicebus.windows.net/;SharedAccessKeyName={PolicyName};SharedAccessKey={Key};EntityPath={ServiceBus}"
+  # - name: consumerID # Optional: defaults to the app's own ID
+  #   value: "{identifier}" 
+  # - name: timeoutInSec # Optional
+  #   value: 60
+  # - name: handlerTimeoutInSec # Optional
+  #   value: 60
+  # - name: disableEntityManagement # Optional
+  #   value: "false"
+  # - name: maxDeliveryCount # Optional
+  #   value: 3
+  # - name: lockDurationInSec # Optional
+  #   value: 60
+  # - name: lockRenewalInSec # Optional
+  #   value: 20
+  # - name: maxActiveMessages # Optional
+  #   value: 10000
+  # - name: maxConcurrentHandlers # Optional
+  #   value: 10
+  # - name: defaultMessageTimeToLiveInSec # Optional
+  #   value: 10
+  # - name: autoDeleteOnIdleInSec # Optional
+  #   value: 3600
+  # - name: minConnectionRecoveryInSec # Optional
+  #   value: 2
+  # - name: maxConnectionRecoveryInSec # Optional
+  #   value: 300
+  # - name: maxRetriableErrorsPerSec # Optional
+  #   value: 10
+  # - name: publishMaxRetries # Optional
+  #   value: 5
+  # - name: publishInitialRetryIntervalInMs # Optional
+  #   value: 500
+```
+
+> __NOTE:__ The above settings are shared across all topics that use this component.
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field              | Required | Details | Example |
+|--------------------|:--------:|---------|---------|
```

Comment:
Looking at the docs here https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-servicebus/ the third column "Details" is narrow in width and makes this page very long to scroll down. Could you make this column wider and "steal" from the other columns in width?

Reply:
Same as above